learn/core_concepts/documents.md

```diff
@@ -38,6 +46,30 @@ In the above example, `"id"`, `"title"`, `"genre"`, `"release-year"`, and `"cast
 Each attribute must be associated with a **value**, e.g. `"Kung Fu Panda"` is the value of `"title"`.
 At minimum, the document must contain one field with the **[primary key][primary-key]** attribute and a unique **[document id][document-id]** as its value. Above, that's: `"id": "1564saqw12ss"`.
 
+#### NDJSON objects
+
+NDJSON objects consist of individual lines where each individual line is valid JSON text and each line is delimited with a newline character. Any [rules that apply to formatting NDJSON](http://ndjson.org/) also apply to MeiliSearch documents.
+
+The above JSON document would look like this in NDJSON:
+
+```ndjson
+
+{ "id": "1564saqw12ss", "title": "Kung Fu Panda", "genre": "Children's Animation", "release-year": 2008, "cast": [ {"Jack Black": "Po"}, {"Jackie Chan": "Monkey"} ]}
+```
+
+#### CSV objects
+
+CSV objects are separated by delimiters that could be a space, semicolon, or some other character, though the comma is most common. Any [rules that apply to formatting CSV](https://datatracker.ietf.org/doc/html/rfc4180) also apply to MeiliSearch documents.
+
+The above JSON document would look like this in CSV:
+
+```csv
+  "id","title","genre","release-year","cast"
+  "1564saqw12ss","Kung Fu Panda","Children's Animation",2008,""
+```
+
+Since CSV does not support arrays or nested objects, the cast cannot be converted to CSV.
```

Comment:
It took a few moments for me to understand what "the cast" was referring to here. 

First of all, "cast" should be marked as code with backticks (`\``). We are not referring to cast, the word, here, but `cast`, the column header, which is in certain ways like a variable name or a an object's key.

Second, that definite article (`the`) is unnecessary. If you want to keep it, you should qualify `cast`: "the `cast` field" could work, for instance.

Reply:
My bad, will update this